tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -422,6 +422,53 @@ DISABLED_DRAW_TEST_F(CGContext, DrawAnImage, UIKitMimicTest) {
     CGContextDrawImage(context, bounds, image.get());
 }
 
+DRAW_TEST_F(CGContext, DrawAnImageWithOpacity, UIKitMimicTest) {
+    // Load an Image and draw it into the canvas context
+    auto drawingConfig = DrawingTestConfig::Get();
+
+    woc::unique_cf<CFStringRef> testFilename{ _CFStringCreateWithStdString(drawingConfig->GetResourcePath("png3.9.png")) };
```

Comment:
>png3.9. [](start = 107, length = 7)

Not too happy about the names here. What is png3.9? A file name should describe at least partially what the file is.